solutions/usaco-1232.mdx

```diff
@@ -0,0 +1,88 @@
+---
+id: usaco-1232
+source: USACO Silver 2022 Open
+title: COW Operations
+author: Mythreya Dharani
+---
+## Explanation
+Note that the characters in any queried string can always be moved around in any order, so the solution might have to do with the parity of the characters. 
+
+Let us define $C$, $O$, and $W$ as the number of respective characters in a substring that is queried. 
```

Comment:
give an example
i.e.
"for example, `COWOW` would have $C=1$, $O=2$, and $W=2$"